source/Posets/Delta-V-CompletePoset.lagda

```diff
@@ -0,0 +1,206 @@
+Ian Ray, 25 July 2023.
+
+Formalizing the auxilary notion of a delta-V-complete poset which is
+sufficient for the main theorems of *link Tom de Jong's thesis*
+involving impredicativity (in the form of resizing axioms) and order theory.
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
+open import MLTT.Two-Properties
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Logic
+
+open import UF.Subsingletons
+open import UF.Subsingletons-FunExt
+
+module Posets.Delta-V-CompletePoset
+ (pt : propositional-truncations-exist)
+ (fe : Fun-Ext)
+  where
+
+open import Locales.Frame pt fe
+
+open import Posets.2ElementPoset pt fe
+
+module δ_𝓥-complete-poset {𝓤 𝓦 : Universe} (𝓥 : Universe) (A : Poset 𝓤 𝓦) where
+ δ : (x y : ∣ A ∣ₚ) → (P : Ω 𝓥) → (𝟙{𝓥} + P holds) → ∣ A ∣ₚ 
+ δ x y P (inl _) = x
+ δ x y P (inr _) = y
+
+ open Joins (rel-syntax A)
+
+ is-δ_𝓥-complete : 𝓤 ⊔ 𝓦 ⊔ (𝓥 ⁺) ̇ 
```

Comment:
I'm not sure about having `_𝓥` as part of the name. The universe `𝓥` is really an argument and just naming it `is-δ-complete` is easier and probably better.